### PR TITLE
WIP: Weave walkthrough with string, bool, None

### PR DESCRIPTION
**Pull request by staceysv**
* adding Shawn's walkthrough notebook
* extending with a quick/basic example of String, Boolean, and None

TODOs/main questions:
1.  Is this correct/expected behavior for the Weave None type? Is there another way to render this panel/access this type? The suggestions in EE seem to be the union of all ops on all types, including 6 instances of the "name" op, which seems off/confusing.
Screen Shot 2023-07-05 at 4 59 10 PM
2. Chainable_ops() no longer works, possible regression?
3. More sophisticated/different examples? 
4. Would be nice to clean up EE suggestions / provide a workaround—I've been writing up general categories & showing examples of those, can also add screenshots.

### CODE REVIEW

**Comment by staceysv** [comment]
extending section on Weave Boolean per @jamie-rasmussen 's recommendation — Weave Nodes with Boolean() type seem to work slightly differently from Weave Numbers and Strings in that you can't operate on the nodes without calling weave.use() first—I illustrate this in the notebook for clarity/completeness, and A) we may want to make this consistent, B) this means the notebook contains a cell that will intentionally error, and this may cause trouble for our automated tests.

**Comment by staceysv** [comment]
addressed additional feedback from @tssweeney 
* replace .chainable_ops() with UI instructions and image until we fix this op
* clarify that the weave fs path is configurable through an env variable
* condense section on Weave Boolean (will move details into more in-depth type-exploration notebook)
* clean up TODOs into last comment cell

File: examples/experimental/skip_test/weave_walkthrough.ipynb [hunk 1]
```diff
@@ -0,0 +1,415 @@
...
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "5f8febdb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import weave\n",
+    "weave.use_frontend_devmode()"
```
Comment (jamie-rasmussen) [inline examples/experimental/skip_test/weave_walkthrough.ipynb:11]: Thinking we want to remove `use_frontend_devmode` before merging?

File: examples/experimental/skip_test/weave_walkthrough.ipynb [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "71ffc2ae",
+   "metadata": {},
+   "source": [
+    "## Intro\n",
```
Comment (jamie-rasmussen) [inline examples/experimental/skip_test/weave_walkthrough.ipynb:19]: This is a nice general overview of Weave, maybe add one sentence explaining what will be covered in this notebook specifically?

File: examples/experimental/skip_test/weave_walkthrough.ipynb [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+   "source": [
+    "weave.use(mynum)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "df1326f6",
+   "metadata": {},
+   "source": [
+    "Weave saves data as a Weave LocalArtifact by default. You can inspect local artifacts at `/tmp/weave/fs/admin/local-artifacts`.\n",
```
Comment (jamie-rasmussen) [inline examples/experimental/skip_test/weave_walkthrough.ipynb:111]: I think it isn't very clear what "data" is - nodes or something else?

Comment (jamie-rasmussen) [inline examples/experimental/skip_test/weave_walkthrough.ipynb:111]: Aside: what's the `admin` in the path represent?

Comment (staceysv) [reply examples/experimental/skip_test/weave_walkthrough.ipynb:111]: I don't know—good question for @tssweeney ?

Comment (staceysv) [reply examples/experimental/skip_test/weave_walkthrough.ipynb:111]: thank you, great point! tried to rephrase like this: Weave saves all of its data—including integers, text, images, and any other user input, as well as any downstream objects produced by computation—as Weave LocalArtifacts by default.

File: examples/experimental/skip_test/weave_walkthrough.ipynb (hunk 2 above)
Comment (staceysv) [reply examples/experimental/skip_test/weave_walkthrough.ipynb:19]: added: This notebook is an evolving introduction to Weave—it walks through the core concepts and existing patterns for composing, manipulating, and extending a compute graph of typed objects.

File: examples/experimental/skip_test/weave_walkthrough.ipynb (hunk 3 above)
Comment (tssweeney) [reply examples/experimental/skip_test/weave_walkthrough.ipynb]: so files are stored at `/tmp/weave/fs/[USER]/` ... when you are authed with W&B on your local machine, your username is `admin`. This is only ever used in the local case

**Review by jlzhao27 — APPROVED** [review]

**Pull request merged**